core/tests/interfaces.rs

```diff
@@ -601,9 +601,11 @@ async fn invalid_fragment() {
         .unwrap();
 
     match &res.to_result().unwrap_err()[0] {
-        QueryError::ExecutionError(QueryExecutionError::UnknownField(_, type_name, field_name)) => {
-            assert_eq!(type_name, "Legged");
-            assert_eq!(field_name, "name");
+        QueryError::ExecutionError(QueryExecutionError::ValidationError(_, error_message)) => {
+            assert_eq!(
+                error_message,
+                "Cannot query field \"name\" on type \"Legged\"."
+            );
```

Comment:
IIRC, I addressed that on the `lutter/gql` branch by accepting either error message

Reply:
Cool, do you want me to drop this change? (because then this branch fails to pass unit tests...) or should i copy it from your branch?

Reply:
For now, I think it would be preferable if tests pass both when `ENABLE_GRAPHQL_VALIDATIONS` is set in the environment and when it is not.